src/tools/base/BaseBrushTool.js

```diff
@@ -18,9 +16,11 @@ const { state, setters } = store.modules.brush;
 class BaseBrushTool extends BaseTool {
   constructor(props, defaultProps = {}) {
     if (!defaultProps.configuration) {
-      defaultProps.configuration = {};
+      defaultProps.configuration = { alwaysEraseOnClick: false };
```

Comment:
Advantage to using this over the merge that takes place in `BaseTool`'s constructor?
Is this to avoid having to do a merge in this BaseBrushTool?